apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore/Trace/CloudTraceExtension.cs

```diff
@@ -138,31 +135,24 @@ public static void UseGoogleTrace(this IApplicationBuilder app)
             var tracerFactory = new ManagedTracerFactory(projectId, consumer,
                 RateLimitingTraceOptionsFactory.Create(config), traceIdFactory);
 
+            services.AddScoped(CreateTraceHeaderContext);
```

Comment:
So this isn't used by anything else, it's just available if you *want* to inject it in a scoped dependency? If we didn't have this line (and the CreateTraceHeaderContext method) would anything else fail?